tests/behat/bootstrap/FeatureContext.php

```
@@ -102,38 +102,34 @@ public function assertInjectedRevealBecomesVisible(): void {
-      menu.appendChild(links);
-      nav.appendChild(toggle);
-      nav.appendChild(menu);
-      document.body.insertBefore(nav, document.body.firstChild);
-      Drupal.attachBehaviors(document.body);
-    ";
-    $session->executeScript($script);
+    $session->executeScript('window.scrollTo(0, 800);');
+    $top = $session->evaluateScript("document.getElementById('siteNav').getBoundingClientRect().top");
+
+    if ((float) $top > 5.0) {
+      throw new \Exception(sprintf('The header is not pinned to the top of the viewport after scrolling (top: %s).', $top));
```

PHPMD (2.15.0)

[error] Missing class import via use statement (line '115', column '17'). (undefined)

(MissingImport)